x/ts/src/spatial/box/box.spec.ts

```diff
@@ -308,29 +308,72 @@ describe("Box", () => {
     });
   });
   describe("contains", () => {
-    it("should return true if the box completely contains the other box", () => {
-      const b = box.construct(0, 0, 20, 20);
-      const b2 = box.construct(5, 5, 15, 15);
-      expect(box.contains(b, b2)).toBe(true);
-    });
-    it("should return false if the box does not completely contain the other box", () => {
-      const b = box.construct(0, 0, 10, 10);
-      const b2 = box.construct(5, 5, 15, 15);
-      expect(box.contains(b, b2)).toBe(false);
-    });
-    it("should return true if the two boxes are equal", () => {
-      const b = box.construct(0, 0, 10, 10);
-      expect(box.contains(b, b)).toBe(true);
-    });
-    it("should return true if the box contains the point", () => {
-      const b = box.construct(0, 0, 10, 10);
-      const p = { x: 5, y: 5 };
-      expect(box.contains(b, p)).toBe(true);
+    describe("inclusive", () => {
+      it("should return true if the box completely contains the other box", () => {
+        const b = box.construct(0, 0, 20, 20);
+        const b2 = box.construct(5, 5, 15, 15);
+        expect(box.contains(b, b2)).toBe(true);
+      });
+      it("should return true if the box completely contains the other box", () => {
+        const b = box.construct(0, 0, 20, 20);
+        const b2 = box.construct(5, 5, 14, 14);
+        expect(box.contains(b, b2)).toBe(true);
+      });
```

Comment:
Don't get the point of this test case from lines 317-321, pretty much a duplicate of the one above.

Reply:
This box sits completely inside, not including the border. The other box sits on the border